NTSB Accident Reports.js

```diff
@@ -0,0 +1,243 @@
+				"institution": "National Transportation Safety Board",
+				"language": "en",
+				"libraryCatalog": "NTSB Accident Reports",
+				"reportNumber": "MAR-21-02",
+				"reportType": "Accident report",
+				"url": "https://www.ntsb.gov/investigations/AccidentReports/Pages/MAR2102.aspx",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					},
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.ntsb.gov/investigations/AccidentReports/Pages/MAB2109.aspx",
+		"items": [
+			{
+				"itemType": "report",
+				"title": "Container Damage and Loss aboard Deck Cargo Barge <i>Ho'omaka Hou</i>, Towed by <i>Hoku Loa</i>​",
+				"creators": [],
+				"date": "2021-04-06",
+				"abstractNote": "​On June 22, 2020, about 0230 local time, the deck cargo barge Ho'omaka Hou was under tow by the towing vessel Hoku Loa off the northeast coast of the big island of Hawaii en route to Hilo, when fifty 40-foot containers stacked on the after deck of the barge toppled, causing 21 to fall into the ocean. There were no injuries or pollution reported. Eight containers were eventually recovered by salvors, and 13 remain missing. Cargo loss was estimated at $1.5 million, and damage to the barge and containers was estimated at $131,000.",
+				"institution": "National Transportation Safety Board",
+				"language": "en",
+				"libraryCatalog": "NTSB Accident Reports",
+				"reportNumber": "MAB-21-09",
+				"reportType": "Accident report",
+				"url": "https://www.ntsb.gov/investigations/AccidentReports/Pages/MAB2109.aspx",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					},
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.ntsb.gov/investigations/AccidentReports/Pages/HWY21MH006-preliminary-report.aspx",
+		"items": [
+			{
+				"itemType": "report",
+				"title": "Preliminary Report - Highway: HWY21MH006",
+				"creators": [],
+				"date": "2021-04-12",
+				"abstractNote": "​The information in this report is preliminary and will be supplemented or corrected during the course of the investigation.",
+				"institution": "National Transportation Safety Board",
+				"language": "en",
+				"libraryCatalog": "NTSB Accident Reports",
+				"reportNumber": "HWY21MH006-preliminary-report",
+				"reportType": "Accident report",
+				"shortTitle": "Preliminary Report - Highway",
+				"url": "https://www.ntsb.gov/investigations/AccidentReports/Pages/HWY21MH006-preliminary-report.aspx",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					},
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					}
```

Comment:
Do we really need both here? Seems like the PDF has the full report.

(Also, this crash, my god.)

Reply:
True. For some reason I thought that the snapshot had images that the PDF lacks, but that's not the case.

And yeah. 23 passengers in an SUV. It's horrible. [The Times' story about it](https://www.nytimes.com/2021/04/04/us/migrants-crash-california-imperial-valley.html) is disturbing and definitely worth a read.